.github/workflows/build.yml

```diff
@@ -27,11 +27,14 @@ jobs:
 
     - name: Install node modules for DemoApp
       working-directory: DemoApp
-      run: yarn install --frozen-lockfile
+      run: yarn install --frozen-lockfile --network-timeout 100000
```

Comment:
I had forgotten why that was "needed". In fact it is only for linting to go through to the DemoApp directory, since it is not ignored  from root eslint. It is indeed an unnecessary step there, as we shouldn't eitherway need DemoApp deps to lint tracker code.